frontend/src/client/services.ts

```diff
@@ -405,6 +407,33 @@ export type TDataUpdateItem = {
 export type TDataDeleteItem = {
   id: string
 }
+export interface TDataCreateInquiry {
+  requestBody: InquiryCreate;
+}
+
+export class InquiriesService {
```

Comment:
I'm curious if we really want these static, since then methods that rely on this function become more difficult to unit test. (Unless I guess you use runtime injection? Thoughts?